php/ui/component/class-plan-details.php

```diff
@@ -79,7 +80,7 @@ protected function plan( $struct ) {
 		}
 
 		$struct['children']['requests'] = $this->make_item( __( 'Total Requests', 'cloudinary' ), number_format_i18n( $data['requests'] ), $this->dir_url . 'css/images/requests.svg' );
-		$struct['children']['assets']   = $this->make_item( __( 'Optimized assets', 'cloudinary' ), $data['resources'], $this->dir_url . 'css/images/image.svg' );
+		$struct['children']['assets']   = $this->make_item( __( 'Optimized assets', 'cloudinary' ), number_format_i18n( Sync_Queue::get_optimized_assets() ), $this->dir_url . 'css/images/image.svg' );
```

Comment:
@pereirinha - When the number is 0, it doesn't show since it checks if the content is empty first.
We can resolve this simply by appending a space to the content on line#155 ( Which will also allow maintaining spacing on smaller screens if there is no data)
```
		$description_part['content']               = $description . '&nbsp;';
```
<img width="258" alt="Screenshot 2022-02-09 at 08 36 01" src="https://user-images.githubusercontent.com/307466/153135352-1f7088b6-ebe9-42b1-a223-e859f00153d2.png">
<img width="279" alt="Screenshot 2022-02-09 at 08 30 45" src="https://user-images.githubusercontent.com/307466/153135371-c1845b25-1352-475d-8d02-a5fe7c63a9f9.png">

Reply:
@pereirinha - A better solution is this. The content render has been bugging me a bit, so this PR addresses it better than the space hack above:
https://github.com/cloudinary/cloudinary_wordpress/pull/685

Reply:
Thanks @DavidCramer